### DISCUSSION TITLE
error oauth

### DISCUSSION THREAD

**Discussion by Rensya23** [PHPMailer/PHPMailer#2911] [General]
Uncaught Error: Interface "PHPMailer\PHPMailer\OAuthTokenProvider" not found in C:\xampp\htdocs\wisata kuliner\PHPMailer\src\OAuth.php:36 Stack trace: #0 C:\xampp\htdocs\wisata kuliner\proses.php(20): require() #1 {main} thrown in C:\xampp\htdocs\wisata kuliner\PHPMailer\src\OAuth.php on line 36
please someone help me

**Comment by Synchro** [comment]
This means that you have not loaded the file, which is probably because you're not using Composer, which deals with loading classes for you. See [the readme for how to load classes](https://github.com/PHPMailer/PHPMailer#installation--loading) (the same things it says about the PHPMailer class apply to OAuthTokenProvider).
